Source/Core/VideoCommon/DriverDetails.cpp

```diff
@@ -58,6 +58,7 @@ namespace DriverDetails
 		{OS_WINDOWS,VENDOR_NVIDIA,   DRIVER_NVIDIA,       -1, BUG_BROKENUNSYNCMAPPING, -1.0, -1.0, true},
 		{OS_LINUX,  VENDOR_NVIDIA,   DRIVER_NVIDIA,       -1, BUG_BROKENUNSYNCMAPPING, -1.0, -1.0, true},
 		{OS_WINDOWS,VENDOR_INTEL,    DRIVER_INTEL,        -1, BUG_INTELBROKENBUFFERSTORAGE, 101810.3907, 101810.3960, true},
+		{OS_ALL,    VENDOR_MESA,     DRIVER_I965,         -1, BUG_BROKENIMAGELOADSTORE,-1.0, -1.0, true},
```

Comment:
This feature is not supported on sandy bridge, so only ivy bridge and newer matters. I've tested it successfully on ivy bridge, likely @Tilka tested it on Haswell. 

I guess this bug is already fixed on all mesa stable releases, so I doubt it's worth to keep this patch at all.